crypto/store/loader_file.c

```diff
@@ -1435,26 +1537,23 @@ static int file_eof(OSSL_STORE_LOADER_CTX *ctx)
 
 static int file_close(OSSL_STORE_LOADER_CTX *ctx)
 {
-    if (ctx->type == is_dir) {
-        OPENSSL_DIR_end(&ctx->_.dir.ctx);
-    } else {
-        BIO_free_all(ctx->_.file.file);
+    if ((ctx->flags & FILE_FLAG_ATTACHED) == 0) {
+        if (ctx->type == is_dir) {
```

Comment:
for the inner conditional statment curl braces should not be used according to the coding style.

Reply:
Fixed